core/src/test/scala/unit/kafka/admin/ResetConsumerGroupOffsetTest.scala

```diff
@@ -416,6 +416,9 @@ class ResetConsumerGroupOffsetTest extends ConsumerGroupCommandTest {
     produceConsumeAndShutdown(topic = topic1, group = group1, totalMessages = 100, numConsumers = 2)
     produceConsumeAndShutdown(topic = topic2, group = group2, totalMessages = 100, numConsumers = 5)
```

Comment:
This test case doesn't really need 5 consumers. Can we change both of these to use a single consumer?